yarn.lock

```diff
@@ -1,1494 +1,1494 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this just seems like noise without any new deps, would you mind reverting this?

Reply:
Done